src/main/webapp/features.jsp

```diff
@@ -2,187 +2,191 @@
 <t:staticPage currentPage="features">
     <div class="container">
         <div class="lead">
-            Here is an overview of some notable TEAMMATES features.
-            The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.
+            <h1 class="color_orange">
+                Features
+            </h1>
+            <p style="font-size:14px"> Here is an overview of some notable TEAMMATES features.
+            The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.</p>
         </div>
         <section class="row">
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 <h2 class="color_orange col-xs-12">Team peer evaluations</h2>
-                <img class="img-responsive col-xs-12" src="images/feature-teampeerevaluations.png" alt="Team peer evaluations">
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    Have a team project in your course? Set up a 'team peer evaluation session' for students to give anonymous peer feedback to team members.
-                    <br><br>
-                    Students can provide confidential peer evaluations to you too.
-                </div>
-                <img class="img-responsive" src="images/raised-edge.png">
+                <img class="img-responsive col-xs-13" src="images/feature-teampeerevaluations.png" alt="Team peer evaluations" style="height: 75%; width: 75%;">
```

Comment:
I think we're using a 12-column grid so there shouldn't be such a class. More info about the Bootstrap grid system here: https://www.w3schools.com/bootstrap/bootstrap_grid_system.asp